matron/src/clocks/clock_link.c

```diff
@@ -91,13 +92,15 @@ static void *clock_link_run(void *p) {
 }
 
 void clock_link_init() {
+    link = abl_link_create(120);
     clock_reference_init(&clock_link_reference);
 }
 
 void clock_link_start() {
     pthread_attr_t attr;
     pthread_attr_init(&attr);
 
+    clock_link_shared_data.number_of_peers = 0;
```

Comment:
it needs to be removed here as well :) please make sure the code builds/runs so it doesn't cause any issues after merging

Reply:
Sure. I'll mark it as a draft for now, since building norns is relatively difficult.